packages/plugin-link-preview/web/src/getToolbarButtons.tsx

```diff
@@ -57,6 +71,7 @@ export const getToolbarButtons = (config, services): ToolbarButton[] => {
       },
       attributes: {
         selectedNode: selectedNodeResolver,
+        visible: isEmbedResolver,
```

Comment:
the name is really confusing - we want to display settings button when linkPreview is selected - 
```suggestion
        visible: isLinkPreviewSelectedResolver,
```